client/state/happychat/user/reducer.js

```diff
@@ -29,4 +29,12 @@ export const geoLocation = createReducer(
 	geoLocationSchema
 );
 
-export default combineReducers( { geoLocation } );
+export const isEligible = createReducer(
```

Comment:
As mentioned above: this code is going to be pulled from Calypso in a few weeks to happychat-client. We aren't using the custom Calypso state utils, but the canonical Redux ones. At the moment, we don't have a schema or a mechanism store the state locally. Can this be refactored to use the canonical Redux utils instead and remove the schema utils as well?